Base/QTGUI/qSlicerApplication.cxx

```diff
@@ -1021,7 +1021,7 @@ void qSlicerApplication::setupFileLogging()
     .arg(tempDir)
     .arg(this->applicationName())
     .arg(qSlicerApplication::application()->applicationVersion())
-    .arg(this->revision())
+    .arg(qSlicerApplication::application()->revision())
```

Comment:
To support custom apps this should be the `mainApplicationRevision`. For regular 3D Slicer `revision` and `mainApplicationRevision` are equivalent, but for Slicer custom apps this will make sure the log file name and corresponding info in the table in the custom app's revision rather than the underlying Slicer version revision.
```suggestion
    .arg(qSlicerApplication::application()->mainApplicationRevision())
```